### DISCUSSION TITLE
Kotlinx Serialization for Server

### DISCUSSION THREAD

**Discussion by j-davies** [ExpediaGroup/graphql-kotlin#1076] [Ideas]
Hello @dariuszkuc,

I was looking through your recent set of changes for kotlin mutli-platform clients to use `kotlinx.serialization`, are there plans to do similar on the server? It would be great to get rid of Jackson completely.

Thanks for all your work on this project.

**Comment by dariuszkuc** [comment]
Hello 👋 
Currently we don't have any plans on moving away from Jackson on the server side.

**Reply by floatdrop** [reply]
@dariuszkuc can you share why this is not planned?

**Reply by dariuszkuc** [reply]
Jackson is arguably the most popular serialization framework in the JVM ecosystem and is the default for Spring. We currently don't have plans on moving away from Jackson (on the server side) so it is unlikely we will be implementing this feature but we are always open to the community contributions.
